### ISSUE TITLE
Video playback choppy after waking from sleep

### ISSUE THREAD

**Issue by irfanhutomo2** [sambow23/Dell-XPS-13-7390-macOS#9]
when i wake from sleep video playback start to choppy, only restart can fix it
i use last EFI Monterey

**Comment by sambow23** [comment]
I just tested myself and it seems fine with Safari on YouTube, even at 4K 60FPS.
What browser/software are you using?

**Comment by sambow23** [comment]
I see you changed the SMBIOS back to MacBookAir9,1 for the video output. Though using that SMBIOS has performance/power management issues. I can try to test USB-C to HDMI/DP on the MacBookPro16,4 SMBIOS, and if it is broken I will look into it

**Comment by irfanhutomo2** [comment]
yes i just try fix my self search on internet,founf on reddit someone say try change SMBIOS, it's work for me.
then what SMBIOS should i use to fix video choppy after sleep? because i user external monitor daily

**Comment by sambow23** [comment]
Try using the `Monterey - Experimental` EFI, it uses the MacBookPro15,4 SMBIOS but also has working USB-C to HDMI/DP. I did not encounter any issues with video playback after waking up from sleep

**Comment by irfanhutomo2** [comment]
okaay i'll try it, thankss mann your the best!!
